client/app/components/_mapPicker/mapPicker.component.js

```diff
@@ -0,0 +1,14 @@
+import template from "./mapPicker.html";
+import controller from "./mapPicker.controller";
+import "./mapPicker.styl";
+
+let mapPickerComponent = {
+  bindings: {
+    placeholder: "@",
+    data: "<" // item with latitude, longitude and address
```

Comment:
You're using this as a two-way binding in the code, aren't you? E.g. when you do `$ctrl.data.something = "bla"` in mapPicker, the parent would see the changes.
So it should be "=" instead of "<". It would still work with "<" (one-way binding) but it violates general angular style guidelines.

Even better than using two-way bindings would be to use output bindings "&", because that is compatible with angular2+

@nicksellen do you have more input on this topic?